src/studio/src/designer/backend/Services/Implementation/ConvertTextsService.cs

```diff
@@ -0,0 +1,109 @@
+using System;
```

Comment:
Same as the controller, this should go in the TextsService that allready is created.